pikaptcha/console.py

```diff
@@ -148,6 +148,9 @@ def entry():
                 ulist.close()
         for x in range(0,args.count):
             print("Making account #" + str(x+1))
+            if (((x+1)%6 = 0)):
```

Comment:
Wow... the `if` statement is always true since you're not actually testing.
`if x%6 == 5:` would be the correct form... but it also doesn't work because the timeout required between 5 accounts is one hour and not 5 minutes.
Maybe this worked for you because it would wait 5 min for every account it created?